sql/internal/specutil/spec.go

```diff
@@ -144,6 +147,31 @@ func QualifyDuplicates(tableSpecs []*sqlspec.Table) error {
 	return nil
 }
 
+// QualifyReferencedTables sets the Qualified field equal to the schema name in any tables
+// that are references across schema boundaries from any foreign key.
+func QualifyReferencedTables(tableSpecs []*sqlspec.Table, realm *schema.Realm) error {
```

Comment:
Please, add simple tests for `postgres/sqlspec_test.go` or `mysql/sqlspec_test.go`.